debian/changelog

```diff
@@ -1,3 +1,17 @@
+citus (9.4.5.citus-1) stable; urgency=low
+  
+  * Adds a configure flag to enforce security
+  
+  * Avoids re-using connections for intermediate results
+  
+  * Fixes a bug that causes pruning incorrect shard of a range distributed table
+  
+  * Fixes a bug that might cause self-deadlocks when COPY used in TX block
+  
+  * Fixes an issue that could cause citus_finish_pg_upgrade to fail
+  
+ -- Gurkan <gindibay@microsoft.com>  Thu, 08 Jul 2021 10:15:23 +0000  
+ 
 citus (10.0.3.citus-1) stable; urgency=low
```

Comment:
See my comment on `citus.spec` file where I shared my take on having 10.x entries in 9.y package changelogs.

Reply:
I put a comment on that comment